### PR TITLE
move nci aws sync dag to its own folder, revise incremental-sync restore dag to perform index

### PR DESCRIPTION
**Pull request by NikitaGandhi**
- move nci aws sync dag to its own folder
- revise incremental-sync restore dag to perform index

### CODE REVIEW

**Comment by NikitaGandhi** [comment]
bringing a change across production

File: dags/nci_aws/k8s_nci_db_incremental_sync.py [hunk 1]
```diff
@@ -67,16 +67,21 @@
...
-        Secret("env", "DB_ADMIN_USER", "explorer-nci-admin", "postgres-username"),
-        Secret("env", "DB_ADMIN_PASSWORD", "explorer-nci-admin", "postgres-password"),
-    ],
 }
 
+# Lift secrets into environment variables for datacube database connectivity
+SECRET_RESTORE_INCREMENTAL_SYNC = [
+    Secret("env", "DB_DATABASE", "explorer-nci-admin", "database-name"),
+    Secret("env", "DB_ADMIN_USER", "explorer-nci-admin", "postgres-username"),
+    Secret("env", "DB_ADMIN_PASSWORD", "explorer-nci-admin", "postgres-password"),
```
Comment (pindge) [inline dags/nci_aws/k8s_nci_db_incremental_sync.py:76]: is this exported by airflow? or intend to?

**Review by pindge — COMMENTED** [review]
would u also like to import `DB_HOSTNAME = "db-writer"` this from variables

File: dags/nci_aws/k8s_nci_db_incremental_sync.py (hunk 1 above)
Comment (NikitaGandhi) [reply dags/nci_aws/k8s_nci_db_incremental_sync.py:76]: yes later

**Review by pindge — APPROVED** [review]

**Pull request merged**